src/Orckestra.Composer.Website/Composite/InstalledPackages/Orckestra.Composer.Product/ManageVariantColors.cshtml

```diff
@@ -210,9 +206,31 @@
                 },
                 showPicker(variant) {
                     this.openedPicker = variant;
+                    if (this.onClick) {
+                        document.removeEventListener('click', this.onClick);
+                    }
+                    this.onClick = (event) => this.documentClick(event, variant);
+                    document.addEventListener('click', this.onClick);
                 },
                 hidePicker() {
                     this.openedPicker = "";
+                    document.removeEventListener('click', this.onClick);
+                    this.onClick = null;
+                },
+                documentClick(e, variant) {
+                    let colorVariants = this.$refs.colorpicker;
+                    let currentTarget = e.target;
+                    let currentColorVariant = colorVariants.find(i => i.id === variant);
+                    let target = currentTarget;
+                    if (currentColorVariant !== currentTarget) {
+                        while (target !== null) {
+                            target = target.parentNode;
+                            if (target && target.id && target.id === currentColorVariant.id) {
+                                return;
+                            }
+                        }
+                        this.hidePicker()
+                    }
```

Comment:
Looks like it can be simple
                    ```
let currentTarget = e.target;
                    let target = currentTarget;
                        while (target !== null) {
                            target = target.parentNode;
                            if (target && target.id && target.id === variant) {
                                return;
                            }
                        }
                        this.hidePicker()
                    
```